main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsAgencySchema.java

```diff
@@ -0,0 +1,35 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+import java.util.Locale;
+import java.util.TimeZone;
+
+
+@GtfsTable("agency.txt")
+public interface GtfsAgencySchema extends GtfsEntity {
+    @FieldType(FieldTypeEnum.ID)
+    @PrimaryKey
+    String agencyId();
+
+    @Required
+    String agencyName();
+
+    @FieldType(FieldTypeEnum.URL)
+    @Required
+    String agencyUrl();
+
+    @Required
+    TimeZone agencyTimezone();
+
+    Locale agencyLang();
```

Comment:
Should this be `@FieldType(FieldTypeEnum.LANGUAGE_CODE)` as well?

Reply:
Ditto.